.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -9,7 +9,146 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository == 'vitejs/vite' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/ecosystem-ci run')
     steps:
-      - uses: actions/github-script@v7
+      - name: Get PR Data
+        uses: actions/github-script@v7
+        id: get-pr-data
+        with:
+          script: |
+            console.log(`Get PR info: ${context.repo.owner}/${context.repo.repo}#${context.issue.number}`)
+            const { data: pr } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            })
+            return {
+              num: context.issue.number,
+              branchName: pr.head.ref,
+              commit: pr.head.sha,
+              repo: pr.head.repo.full_name
+            }
+
+      - name: Check Package Existence
+        uses: actions/github-script@v7
+        id: check-package
+        with:
+          script: |
+            const prData = ${{ steps.get-pr-data.outputs.result }}
+            const url = `https://pkg.pr.new/vite@${prData.commit}`
+            const response = await fetch(url)
```

Comment:
We do not yet implement the `HEAD` for pkg.pr.new! So here instead of consuming the request body and waiting for it, we just check the status and then the step would be done. 

So if it was fetching, it'd be aborted.

Reply:
cc @dominikg

Reply:
if you await fetch(), the full file has been downloaded already, its just not been parsed to json yet. of course if pkg.pr.new does not implement head we have to go with full, but should be considered as an optimization. or do you implement the endpoints about package information that the public registry has?

Reply:
hmm, maybe need to get more coffee and you are right.

Reply:
Haha, thanks, let me get my own coffee too now 😅

Reply:
> We do not yet implement the `HEAD` for pkg.pr.new!

Any plan to support it? It sounds interesting.

Reply:
> Any plan to support it? It sounds interesting.

I'll create an issue for it then!